### DISCUSSION TITLE
Are there any webmock 3.14.0 incompatibilities?

### DISCUSSION THREAD

**Discussion by dogweather** [vcr/vcr#922] [Q&A] [answered]
I'm getting `wrong number of arguments (given 1, expected 0)` with VCR **6.0.0** and Webmock **3.14.0** when I `c.hook_into :webmock`:

```
/Users/robb/.rbenv/versions/3.1.0/bin/bundle: warning: Exception in finalizer #<Proc:0x000000010f521290 /Users/robb/.rbenv/versions/3.1.0/lib/ruby/gems/3.1.0/gems/vcr-6.0.0/lib/vcr/library_hooks/webmock.rb:36 (lambda)>
/Users/robb/.rbenv/versions/3.1.0/lib/ruby/gems/3.1.0/gems/vcr-6.0.0/lib/vcr/library_hooks/webmock.rb:36:in `block in global_hook_disabled_requests': wrong number of arguments (given 1, expected 0) (ArgumentError)
```

Just wondering if this is a known problem before I submit a bug report Issue.

**Comment by olleolleolle** [comment, marked as answer]
This has been fixed in https://github.com/vcr/vcr/pull/907. A new release has not been made. The test suite does not pass green with the latest major release of Faraday, currently. 

Workaround: use the unreleased code, `gem "vcr", github: "vcr/vcr"` in your Gemfile?

Hope this helps!

**Reply by dogweather** [reply]
Thanks, I'll give it a try.

**Reply by dogweather** [reply]
Excellent - thank you.
